eng/Packages.Data.props

```diff
@@ -152,7 +152,7 @@
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions.Rpc" Version="3.0.37" />
     <PackageReference Update="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0" />
     <PackageReference Update="Microsoft.Spatial" Version="7.5.3" />
-    <PackageReference Update="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Are we confident we don't have any shared-environment dependencies related to this package, e.g. from PowerShell, Functions, or other, where if they loaded an earlier version first, it could break end-users in those environments?  If we've done our due-diligence around this, I have no concerns, and thanks for making this happen, @m-nash!

Reply:
Ah, just noticed @JoshLove-msft already addressed from a functions perspective.  Thanks!

Reply:
@annelo-msft  who can we loop in to validate that I wouldn't know where to start.

Reply:
I think functions and PowerShell are the only two we track for Core - @jsquire was working on tracking a dependency down for PowerShell and may be able to share the contact there.  If not, I can dig up the name Pavel gave us when he handed this off.

Reply:
I did an investigation with PowerShell compatibility when I updated `System.Diagnostics.DiagnosticSource`. I contacted a handful of people that I could find, and it was hard for me to get any kind of authoritative answer (If I even got an answer at all :sweat_smile:). However, the consensus of what I did find out was that as long as it's not higher than the version in PowerShell it shouldn't be a problem.

Looking at their changelog it looks like they bumped to 13.0.1 in 7.2 - https://github.com/PowerShell/PowerShell/blob/master/CHANGELOG/7.2.md#build-and-packaging-improvements-20

I don't know exactly how their dependencies are set up, but it seems like the current reference in 7.4 is 13.0.3 - https://github.com/PowerShell/PowerShell/blob/master/tools/cgmanifest.json#L235 / https://github.com/PowerShell/PowerShell/blob/master/ThirdPartyNotices.txt#L725

Reply:
This also backs that up: https://github.com/Azure/azure-sdk-for-net/issues/8739#issuecomment-552984338

Reply:
Amazing - thanks for figuring that out, @m-redding!!